### DISCUSSION TITLE
Coderunner seems to be misused for spamming

### DISCUSSION THREAD

**Discussion by Silassentinel** [community/community#101685] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

First i received a spam/phising email from https://github.com/MichalZalecki/
see attached images 
![Selected photo (1).jpeg](https://github.zendesk.com/attachments/token/jrvI74bRv0h8SXQkGl3wk7kSN/?name=Selected+photo+%281%29.jpeg)
![Selected photo](https://github.com/community/community/assets/53788992/05a9b74c-d4a2-493a-92b9-b845a7e93883)
I went to google to find out who this person was, found this website: https://michalzalecki.com/

Which links to this persons github page.

There i did a search on the term cruiser and found the repo which is mentioned in the mail.

![Selected photo (1)](https://github.com/community/community/assets/53788992/d04ccc5b-9b7a-4749-8a00-1802547a6d1a)

![Selected photo](https://github.com/community/community/assets/53788992/cb7b6fac-23d2-4de6-92c0-3bf9ba596250)

NEXT is saw that there were notifications pending , i clicked then saw participating ones , there i saw the same REPO and also another.

![participationNotification](https://github.com/community/community/assets/53788992/272a15c9-8f14-4286-9053-b57493cb145d)

This make me suspect that the coderunner or the function being used is running abnormal!

Also i have in my profile my email address marked as private so how is it possible my personal email adres  leaked???

**Comment by samus-aran** [comment, marked as answer]
Hi @Silassentinel 👋 

We've seen some abuse like this and our security teams are working hard on stopping this sort of activity in its tracks so that we can keep our users from having to witness content like this. Unfortunately some of it makes it through the notifications system before we get it off the site but we're working on shoring this up too. (You might find by the time you visit the repo in question the content is nowhere to be found!)

I'd encourage anyone who receives an email similar to this to ignore it and do not click any links. Users are always free to report abuse on the platform by following these instructions:

[Reporting abuse or spam](https://docs.github.com/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
